### PR TITLE
db assignment 1

### PR DESCRIPTION
**Pull request by b4ym4xjr**

### CODE REVIEW

File: database/assignment-1/script.sql [hunk 1]
```diff
@@ -1,154 +1,178 @@
...
-    comment_id INT NOT NULL,
-    Foreign Key (comment_id) REFERENCES comments(id) ON DELETE NO ACTION,
-    reply TEXT NOT NULL,
-    created_at TIMESTAMP DEFAULT NOW()
-);
-
-CREATE TABLE post_metadata(
-    post_id INT NOT NULL,
+CREATE TABLE metadata(
+    id SERIAL PRIMARY KEY,
```
Comment (manishgs) [inline database/assignment-1/script.sql:59]: still not scalable , what if we want to add new property of a post how do you handle it?

File: database/assignment-1/script.sql [hunk 2]
```diff
@@ -1,154 +1,178 @@
...
-    (1, 1, 'this is a comment');
+    (1, 'post1 title', 'post1 content', 'draft', 1);
 
+-- add tags to a post
 INSERT INTO
-    replies(comment_id, reply)
+    post_tags(post_id, tag_id)
 VALUES
-    ('1', 'this is a reply');
+    (1, 1);
```
Comment (manishgs) [inline database/assignment-1/script.sql]: add multiple tags

File: database/assignment-1/script.sql [hunk 3]
```diff
@@ -1,154 +1,178 @@
...
 
+-- create metadata for a post
 INSERT INTO
```
Comment (manishgs) [inline database/assignment-1/script.sql:112]: what if featured is already exists in metadata table?

File: database/assignment-1/script.sql [hunk 4]
```diff
@@ -1,154 +1,178 @@
...
 
-SELECT
-    posts.title,
-    posts.content,
-    tags.name
-FROM
-    posts
-    INNER JOIN tags ON tags.name = 'tag1';
+-- increase views
+UPDATE
```
Comment (manishgs) [inline database/assignment-1/script.sql:171]: what if post_id=1 is not exists?

File: php/assignment-2/magic_method.php [hunk 5]
```diff
@@ -0,0 +1,29 @@
+<?php
```
Comment (manishgs) [inline php/assignment-2/magic_method.php:1]: This PR should include changes related to database assignment

**Comment by b4ym4xjr** [comment]
db assignment

**Pull request closed without merging**